### DISCUSSION TITLE
SOFT/GX configs with net emulator.

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/webMAN-MOD#660] [General]
Looking at the wiki entries for available config commands, it seems that GX and SOFT configs use very different command numbers to what the final NET emulator uses. I wonder if it is truly beneficial to include and use these extracted configs with the NET emulator, since I would expect most commands to be corrupted. Or are soft/gx configs included for some other purpose? (it was my understanding that it is only possible to load configs into the net emulator).

As as aside, I was also thinking it would be beneficial to save custom configs in txt format as well, with one command+arguments per line. That way, git history would be able to very clearly show any changes done to them. Though, doing this with the current database would be a huge amount of work requiring going over git history and parsing commands into txt at each stage, so it isn't really a worthwhile effort.

**Comment by aldostools** [comment]
Dear @rhynec 
Thank you for your feedback. I'm not an expert about PS2 CONFIGs. The database is mainly maintained by @agrippa and @mrjaredbeta. I only repack their posts and put it online. From their posts I understand that GX and SOFT configs are usable on ps3_netemu. If not, then these CONFIGs can be used as reference.

You can ask them in https://www.psx-place.com/threads/compatibility-list-ps2-on-ps3.1306/

In regards to the text version of the configs, they are posted in that format in the link above.
